File: PooCC01CNetz/Semana02/Java/Ejercicios3/Automovil.java
// Clase
public class Automovil {
    // Atributos
    public String modelo;
    public double precio;
    public int anio;

    // Constructor
    public Automovil(String modelo, double precio){
        this.modelo = modelo;
        this.precio = precio;
        this.anio = 2023;
    }

    public Automovil() {}

    // Metodo
    public boolean verifica(int anioAct) {
        int verif = anioAct - anio;
        if (verif < 2)
            return false;
        return true;
    }
}

File: PooCC01CNetz/Semana02/Python/Ejercicios3/Automovil.py
class Automovil:
    # Constructor
    def __init__(self, modelo="", precio=0):
        self.modelo = modelo
        self.precio = precio
        self.anio = 2023

    # Metodo
    def verifica(self, anio_act):
        verif = anio_act - self.anio
        if verif < 2:
            return False
        return True
